timemachine/cpp/CMakeLists.txt

```diff
@@ -84,6 +84,7 @@ pybind11_add_module(${LIBRARY_NAME} SHARED NO_EXTRAS
   src/summed_potential.cu
   src/fanout_summed_potential.cu
   src/device_buffer.cu
+  src/kernels/k_lambda_transformer.cu
```

Comment:
May also need to modify another line in CMakeLists.txt (I think we can revert https://github.com/proteneer/timemachine/pull/403/files#diff-91d026c1fcb8424ad567b2343d63101fc584eb17536930067d7241f94ba7ae7eL63-R63 ?)

Reply:
Good call, removed in 5829a43. Also removed `cudart` and `cudadevrt`, which seem to be unnecessary, in 4e1e72